prow/config.yaml

```diff
@@ -1106,6 +1106,29 @@ periodics:
     - name: test-account
       secret:
         secretName: test-account
+- cron: "0 1  * * *"
```

Comment:
Nice idea! Only a pity we have to run this on old Tekton, but it shouldn't matter much for this pipeline